doc/classes/PhysicsServer3D.xml

```diff
@@ -1014,10 +1014,261 @@
 			<description>
+				Returns the bounds of a soft body in global coordinates.
+			</description>
+		</method>
+		<method name="soft_body_get_collision_layer" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the physics layer or layers a soft body belongs to.
+			</description>
+		</method>
+		<method name="soft_body_get_collision_mask" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the physics layer or layers a soft body can collide with.
+			</description>
+		</method>
+		<method name="soft_body_get_damping_coefficient" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the damping coefficient of a soft body.
+			</description>
+		</method>
+		<method name="soft_body_get_drag_coefficient" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the drag coefficient of a soft body.
+			</description>
+		</method>
+		<method name="soft_body_get_linear_stiffness" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the linear stiffness of a soft body.
+			</description>
+		</method>
+		<method name="soft_body_get_point_global_position" qualifiers="const">
+			<return type="Vector3" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="point_index" type="int" />
+			<description>
+				Returns the current position of a soft body point at the specified index in global coordinates.
+			</description>
+		</method>
+		<method name="soft_body_get_pressure_coefficient" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the pressure coefficient of a soft body.
+			</description>
+		</method>
+		<method name="soft_body_get_simulation_precision" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the simulation precision of a soft body.
+			</description>
+		</method>
+		<method name="soft_body_get_space" qualifiers="const">
+			<return type="RID" />
+			<param index="0" name="body" type="RID" />
+			<description>
+				Returns the [RID] of the space assigned to a soft body.
+			</description>
+		</method>
+		<method name="soft_body_get_state" qualifiers="const">
+			<return type="Variant" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="state" type="int" enum="PhysicsServer3D.BodyState" />
+			<description>
+				Returns a soft body state (see [enum BodyState] constants).
+				[b]Note:[/b] Godot Physics does not support [constant BODY_STATE_LINEAR_VELOCITY], [constant BODY_STATE_ANGULAR_VELOCITY], [constant BODY_STATE_SLEEPING], or [constant BODY_STATE_CAN_SLEEP].
```

Comment:
May be worth specifying that Godot Physics is the default physics implementation. (like you did in `soft_body_set_drag_coefficient`)

Reply:
Yes, good point.